package.json

```diff
@@ -70,6 +70,7 @@
     "@typescript-eslint/eslint-plugin": "^4.26.1",
     "@typescript-eslint/parser": "^4.26.1",
     "copyfiles": "^2.3.0",
+    "dayjs": "^1.11.6",
```

Comment:
Do we really need to add this dependency? It's only used in a single example and we could probably put something else there :smile:

Reply:
I don't know why this was here in Lime Elements. @adrianschmidt do you know.

Reply:
Yes. It's used to format the timestamp for when the event was printed. We could use `Intl.DateTimeFormat` instead. I can fix, but not right now.

Reply:
is it so important to format the timestamp?

Reply:
I'm literally taking time away from doing to respond to this question. Just let me do it, ok?

Reply:
⚡

Reply:
The top one is with formatting, the bottom one is without:

<img width="525" alt="image" src="https://user-images.githubusercontent.com/154725/215484677-de4af96e-218c-4985-82bf-0b09c75d5656.png">

Reply:
for me, it's absolutely fine without formatting. but you decide.

Reply:
or can the date be moved inside the `<pre`? To make the header shorter?